SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/PlayerDisconnect/PlayerDisconnectController.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "PlayerDisconnectController.h"
+#include "InputCoreTypes.h"
+
+void APlayerDisconnectController::SetupInputComponent()
+{
+	Super::SetupInputComponent();
+	EnableInput(this);
+	check(InputComponent);
+	InputComponent->BindKey(EKeys::M, IE_Pressed, this, &APlayerDisconnectController::ReturnToMainMenu);
```

Comment:
Is this because it's also used in the manual gym?

Reply:
Yes